### ISSUE TITLE
DeadCodeRule false positive on switch statements

### ISSUE THREAD

**Issue by lqi** [oclint/oclint#24]
`case 3` in the following code is reported as dead code, but it's a false positive.

```
switch (option)
{
    case 1:
        doSomething();
        break;
    case 2:
        doFoo();
        return -1;
    case 3:
        doBar();
        return -2;
    ...
}
```
